project_task_category/views/project_task_view.xml

```diff
@@ -46,5 +46,14 @@
                 </field>
             </field>
         </record>
+        <act_window id="action_type_category_project_task"
+            name="Project category"
```

Comment:
@victormartinelicocorp AFAIK OCA guideline is to align attributes with the one from the first line, right?